samples/snippets/cloud-client/src/test/java/privateca/SnippetsIT.java

```diff
@@ -77,24 +74,24 @@ public static void reqEnvVar(String envVarName) {
   }
 
   @BeforeClass
-  public static void setUp() throws InterruptedException, ExecutionException, IOException {
+  public static void setUp()
+      throws IOException, ExecutionException, NoSuchProviderException, NoSuchAlgorithmException,
+          InterruptedException {
     reqEnvVar("GOOGLE_APPLICATION_CREDENTIALS");
     reqEnvVar("GOOGLE_CLOUD_PROJECT");
 
     LOCATION = "asia-south1";
-    KMS_LOCATION = "global";
     CA_POOL_NAME = "ca-pool-" + UUID.randomUUID().toString();
     CA_POOL_NAME_DELETE = "ca-pool-" + UUID.randomUUID().toString();
     CA_NAME = "ca-name-" + UUID.randomUUID().toString();
     CA_NAME_DELETE = "ca-name-" + UUID.randomUUID().toString();
     CERTIFICATE_NAME = "certificate-name-" + UUID.randomUUID().toString();
-    KEY_RING_ID = "key-ring-id-" + UUID.randomUUID().toString();
-    KEY_ID = "key-id-" + UUID.randomUUID().toString();
-    VERSION_ID = "1";
+    KEY_SIZE = 2048;
 
     // Create CA Pool.
     privateca.CreateCaPool.createCaPool(PROJECT_ID, LOCATION, CA_POOL_NAME);
     privateca.CreateCaPool.createCaPool(PROJECT_ID, LOCATION, CA_POOL_NAME_DELETE);
```

Comment:
what is this used for?

Reply:
(and same for CA_NAME_DELETE on line 102-103)

Reply:
I wanted to specify the default key size as 2048, to maintain the standard value across our tests.

Reply:
CA_NAME_DELETE is entirely required only for testing purposes as all the tests in this repo run concurrently.
Essentially, we perform create and delete test operations at the same time and hence would require more than one CA.

Reply:
CA_POOL_NAME_DELETE is being used to test CA deletion. Kindly refer to line number 218-219.
This is essential to demonstrate the deletion behaviour.

Kindly let me know if you think this approach could be simplified in some other way! Thanks!

Reply:
ah I see, I didn't realize there were a bunch of tests at the bottom that exercise various code paths. Fair enough then, feel free to resolve =)